packages/browser-integration-tests/standalone-custom-key.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
```

Comment:
Following the pattern of the other integration tests -- can we move this to standalone-playground?

Reply:
This is as per how @chrisradek advised me to setup an integration test